src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindBreakpointEvidenceSpark.java

```diff
@@ -477,40 +485,82 @@ public AlignedAssemblyOrExcuse apply( final Tuple2<Integer, List<SVFastqUtils.Fa
     }
 
     /**
+     * For a set of interesting kmers, count occurrences of each over all reads, and remove those
+     * that appear too frequently from the set.
+     */
+    private static HopscotchUniqueMultiMap<SVKmer, Integer, KmerAndInterval> removeUbiquitousKmers(
+            final FindBreakpointEvidenceSparkArgumentCollection params,
+            final JavaSparkContext ctx,
+            final HopscotchUniqueMultiMap<SVKmer, Integer, KmerAndInterval> kmersAndIntervals,
+            final JavaRDD<GATKRead> unfilteredReads,
+            final SVReadFilter filter,
+            final Logger logger ) {
+        final Broadcast<HopscotchUniqueMultiMap<SVKmer, Integer, KmerAndInterval>> broadcastKmersAndIntervals =
+                ctx.broadcast(kmersAndIntervals);
+
+        final int kmersPerPartition = params.cleanerKmersPerPartitionGuess;
+        final int kSize = params.kSize;
+        final int maxQNamesPerKmer = params.maxQNamesPerKmer;
+        final List<SVKmer> ubiquitousKmers =
+                unfilteredReads
+                        .filter(filter::notJunk)
+                        .filter(filter::isPrimaryLine)
+                        .mapPartitions(readItr ->
+                            new KmerCounter(kSize,kmersPerPartition,broadcastKmersAndIntervals.getValue()).apply(readItr))
+                        .mapToPair(kmerAndCount -> new Tuple2<>(kmerAndCount.getKey(), kmerAndCount.getValue()))
+                        .reduceByKey(Integer::sum)
+                        .filter(pair -> pair._2() > maxQNamesPerKmer)
```

Comment:
do you mean `<=` instead of `>` for the per kmer qname count filter?

Reply:
No, this is right.  I'm producing a list of ubiquitous kmers which I eliminate from the list of interesting kmers a few lines below here.